packages/nextjs/scripts/buildRollup.ts

```diff
@@ -1,24 +0,0 @@
-import * as childProcess from 'child_process';
-import * as fs from 'fs';
-import * as path from 'path';
-
-/**
- * Run the given shell command, piping the shell process's `stdin`, `stdout`, and `stderr` to that of the current
- * process. Returns contents of `stdout`.
- */
-function run(cmd: string, options?: childProcess.ExecSyncOptions): string | Buffer {
-  return childProcess.execSync(cmd, { stdio: 'inherit', ...options });
-}
-
-run('yarn rollup -c rollup.npm.config.mjs');
-
-// Regardless of whether nextjs is using the CJS or ESM version of our SDK, we want the code from our templates to be in
```

Comment:
The nextjs integration failures are likely to be related this change...

Reply:
Yes please keep this script!